docs/references.rst

```diff
@@ -0,0 +1,3 @@
+=====================================================
```

Comment:
What sort of thing is going to go here?

Reply:
Important links, e.g.: knowledge base, forum; links to other libraries (dependencies)

Reply:
Let's dump some links in here as a priority then - any low hanging fruit should be tackled quickly so we can assess if it's the best place for that information.